tools/devops/device-tests-provisioning.csx.in

```diff
@@ -5,8 +5,14 @@ using System.Linq;
 
 using static Xamarin.Provisioning.ProvisioningScript;
 
-var xcodePath = Path.GetDirectoryName (Path.GetDirectoryName ("@XCODE_DEVELOPER_ROOT@"));
-Xcode ("@XCODE_VERSION@").XcodeSelect (allowUntrusted: true, allowSymlink: true, symlinkTarget: xcodePath);
+var xcodeVersion = new Version ("@XCODE_VERSION@");
+
+if (xcodeVersion.Build == -1) { // we are dealing with a .0 release, Version ("11.4.3") returns 3, Version ("11.4.0") returns 0 and Version ("11.4") returns -1
```

Comment:
super nit: maybe add a comment re how provisionator expects blah.blah.blah and blah.blah is invalid? This is already kind of in your comment but could be good to have some clear examples of what type of build # format provisionator expects.